src/components/sidebar/components/sidebar-nav-highlight-item/index.tsx

```diff
@@ -33,7 +33,9 @@ export default function SidebarNavHighlightItem({
 			className={classNames(s.root, s[`theme-${theme}`])}
 			href={href}
 		>
-			{isProductSlug(theme) ? <ProductIcon productSlug={theme} /> : null}
+			{isProductSlug(theme) ? (
+				<ProductIcon className={s.icon} productSlug={theme} />
+			) : null}
```

Comment:
Added `flex-shrink: 0` to this icon to prevent the icon shrinking when the `text` is this component is long (as it is for the title of the HCP Vault Secrets `schema.info.title` value).